cmd/security-bootstrapper/res/configuration.yaml

```diff
@@ -1,4 +1,6 @@
 LogLevel: INFO
+Database:
+  Type: postgres
```

Comment:
remove this config
just reuse the Database config under StageGate
The value should be replaced by the environment variables

Reply:
Done